cpp/src/io/parquet/writer_impl.cu

```diff
@@ -1160,21 +1163,21 @@ void writer::impl::write(table_view const& table)
     init_page_fragments(fragments, col_desc, num_rows, max_page_fragment_size);
   }
 
-  size_t global_rowgroup_base = md.row_groups.size();
+  auto const global_rowgroup_base = static_cast<size_type>(md.row_groups.size());
```

Comment:
You could rename `s/rowgroup/row_group/g` in this file for consistency if you want.

Reply:
@vuule This is marked as `solved` but I still see a number of `rowgroup` namings in this file. Is this intended?